e2e-tests/utils.ts

```diff
@@ -208,6 +203,13 @@ export async function loadLocalGraph(page: Page, path?: string): Promise<void> {
 
   await page.waitForFunction('window.document.title === "Logseq"')
 
+  // If there is an error notification from a previous test graph being deleted,
```

Comment:
I like that this makes local development easier by ignoring this intermittent error but I also don't understand why the error occurs. 
@andelf @cnrpman Is it ok to ignore this error that intermittently occurs?:
```
The directory /some/path/logseq/tmp/e2e-test/9unSDE1CXs has been renamed or deleted, the editor will be disabled for this graph, you can unlink the graph.
```
Do we know why it occurs?

Reply:
There are some remnants about the graph used in the previous E2E run in the Local storage I guess.